x-pack/plugins/security_solution/cypress/e2e/detection_alerts/changing_alert_status.cy.ts

```diff
@@ -194,6 +197,7 @@ describe('Changing alert status', () => {
           cy.get(SELECTED_ALERTS).should('have.text', `Selected ${numberOfAlertsToBeOpened} alert`);
 
           openAlerts();
+          cy.wait(10000);
```

Comment:
Hardcoded waits are not allowed, please find another solution or ask for support on this.

Reply:
Thanks for pointing it out.. it is a mistake.. I will remove this.